homework/fibonacci/fibonacci.hpp

```diff
@@ -1,11 +1,24 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+  if (sequence != 0) {
+    auto b{1};
+    for (auto i = 1, a = 0; i < sequence; i++) {
+      b += a;
+      a = b - a;
+    }
+    return b;
+  }
+  return 0;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
+
+  if (sequence == 0) {
     return 0;
+  } else if (sequence == 1) {
+    return 1;
```

Comment:
Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`